### ISSUE TITLE
Bug: curl does not display correct Response Content Type

### ISSUE THREAD

**Issue by rick22sun** [swagger-api/swagger-ui#1397]
This is the result of swagger forum discussion "How to show Response Content Type?".
In summary, with any non-empty schema in a ‘successful’ (2XX) response, e.g, 
```json
{"tags": ["mytag"],
        "produces": ["application/json","application/xml"],
        "responses": {
            "200": {"description": "Success", "schema": {"type" : "integer"}},
            "405": {"description": "Invalid input"}  }};
```
"Response Content Type" pops up in ui, but changing it to the 2nd value does not update curl command display after "Try it out". This is only about curl display, contents sent to server might be correct.

**Comment by iushankin** [comment]
This issue depends on swagger-api/swagger-js#504

**Comment by webron** [comment]
This is resolved in 2.1.1.
